docs/csharp/whats-new/csharp-6.md

```diff
@@ -351,36 +351,29 @@ can be placed between the curly braces of an interpolated string.
 
 ### String interpolation and specific cultures
 
-All the examples shown in the preceding section will format the strings using the current
+All the examples shown in the preceding section format the strings using the current
 culture and language on the machine where the code executes. Often you
 may need to format the string produced using a specific culture.
-The object produced from a string interpolation is a type that has an
-implicit conversion to either <xref:System.String> or <xref:System.FormattableString>.
+To do that use the fact that the object produced by a string interpolation can be implicitly converted to <xref:System.FormattableString>.
 
-The <xref:System.FormattableString> type contains the format string, and the results
-of evaluating the arguments before converting them to strings. You can
+The <xref:System.FormattableString> instance contains the format string, and the results
+of evaluating the expressions before converting them to strings. You can
 use public methods of <xref:System.FormattableString> to specify the culture when
-formatting a string. For example, the following will produce a string
-using German as the language and culture. (It will use the ',' character
+formatting a string. For example, the following example produces a string
+using German culture. (It uses the ',' character
 for the decimal separator,
 and the '.' character as the thousands separator.)
 
 ```csharp
 FormattableString str = $"Average grade is {s.Grades.Average()}";
-var gradeStr = string.Format(null, 
-    System.Globalization.CultureInfo.CreateSpecificCulture("de-de"),
-    str.GetFormat(), str.GetArguments());
+var gradeStr = str.ToString(System.Globalization.CultureInfo.GetCultureInfo("de-DE"));
 ```
 
 > [!NOTE]
 > The preceding example is not supported in .NET Core version 1.0.1. It is
```

Comment:
@rpetrusha @BillWagner is this note still true? What version of .NET Core does support the example?

Reply:
As far as I can tell:

* No version of .Net Framework or .Net Core supports the weird version of `string.Format` or `FormattableString.GetFormat()` that you replaced. (Maybe it's based on some preview version?)
* .Net Core 1.0 does not support `CultureInfo.CreateSpecificCulture` or `CultureInfo.GetCultureInfo`. If you switch to `new System.Globalization.CultureInfo("de-DE")`, the code above will work on .Net Core 1.0+ and .Net Framework 4.6+ (which is the version that introduced `FormattableString`).

Reply:
@svick thank you for clarification, I've updated the example.